src/main/java/goodgenerator/loader/NaquadahReworkRecipeLoader.java

```diff
@@ -58,25 +63,26 @@ public static void RecipeLoad() {
                             Materials.Gallium.getDust(64)
                         },
                         new int[] {2500, 2500, 2500, 2500},
-                        20 * 20,
-                        (int) GT_Values.VP[9],
+                        10 * 20,
+                        (int) GT_Values.VP[10],
                         2);
                 // Enriched Naquadah (UIV)
                 CORE.RA.addQuantumTransformerRecipe(
                         new ItemStack[] {
                             enrichedNaquadahEarth.get(OrePrefixes.dust, 32),
                             Materials.Zinc.getDust(64),
+                            Materials.Carbon.getDust(1),
                             GT_Utility.copyAmount(0, GenericChem.mAdvancedNaquadahCatalyst)
                         },
-                        new FluidStack[] {Materials.SulfuricAcid.getFluid(16000), P507.getFluidOrGas(10)},
+                        new FluidStack[] {Materials.SulfuricAcid.getFluid(16000), Materials.Oxygen.getGas(100L)},
                         new FluidStack[] {wasteLiquid.getFluidOrGas(32000)},
                         new ItemStack[] {
                             inertEnrichedNaquadah.get(OrePrefixes.dust, 64),
                             Materials.Trinium.getDust(64),
                             ItemList.NaquadriaSupersolid.get(1)
                         },
                         new int[] {3300, 3300, 3300},
-                        20 * 20,
+                        10 * 20,
                         (int) GT_Values.VP[11],
                         3);
```

Comment:
Can be made to tier 2 IMO. Enriched naq isn't used a lot

Reply:
Done.